static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -16,7 +16,7 @@ import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import type {Group} from 'sentry/types/group';
 import {trackAnalytics} from 'sentry/utils/analytics';
-import type {FeedbackIssue} from 'sentry/utils/feedback/types';
+import type {FeedbackIssueListItem} from 'sentry/utils/feedback/types';
```

Comment:
This was the 'wrong' type before (ok, `FeedbackIssue` and `FeedbackIssueListItem` were the same, but using the right name matters)